### PR TITLE
Improves url widget

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**

### CODE REVIEW

File: collect_app/src/main/res/values/style.xml [hunk 1]
```diff
@@ -1,6 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
+    <style name="BaseWidgetStyle">
+        <item name="android:layout_marginLeft">7dp</item>
+        <item name="android:layout_marginTop">5dp</item>
+        <item name="android:layout_marginRight">7dp</item>
+        <item name="android:layout_marginBottom">5dp</item>
+        <item name="android:padding">16dp</item>
```
Comment (lognaturel) [inline collect_app/src/main/res/values/style.xml]: I believe the other widget buttons currently have 8dp padding. Does that sound right?

Comment (shobhitagarwal1612) [reply collect_app/src/main/res/values/style.xml]: I have used the margin values that were already being used in this widget

Comment (lognaturel) [reply collect_app/src/main/res/values/style.xml]: And what about the padding? That button looks a lot taller than other widget buttons to me!

Comment (shobhitagarwal1612) [reply collect_app/src/main/res/values/style.xml]: Let me check once again

Comment (shobhitagarwal1612) [reply collect_app/src/main/res/values/style.xml]: Yes, you are right. This was updated after the merging of the last PR. I have added the `dimens.xml` for removing hard coded values in #1176.

**Pull request merged**